accepted/build/references-into-frameworkreferences.md

```diff
@@ -0,0 +1,59 @@
+<Reference Include="PresentationFramework.Luna" Aliases="Luna" />
+```
+
+More discussion of the WPF Theme assemblies (and the inspiration for this proposal) is here: https://github.com/dotnet/sdk/issues/3265
+
+### Turn off default references to assemblies in targeting pack
+
+- Projects in CoreFx [would like](https://github.com/dotnet/sdk/issues/3295) to have a `FrameworkReference` to Microsoft.NETCore.App in order to get the App Host and the right data in the runtimeconfig.json file.  However, they can't include the normal references from the targeting pack, as they are building projects which are part of the targeting pack.
+- WPF wants to explicitly choose all their references, in order to avoid accidentally taking a dependency on APIs such as LINQ.
+- Libraries might want to reference specific assemblies in a shared framework without exposing the whole shared framework as API surface to their consumers.
+
+The project would include something like this:
+
+```xml
+<FrameworkReference Update="Microsoft.WindowsDesktop.App" IncludeReferences="False" />
+<Reference Include="System.Printing" />
+```
+
+## Implementation details
+
+### ResolveAssemblyReference
+
+The ResolveAssemblyReference task would be updated to understand the "new-style" targeting packs, and the FrameworkList files in them.  There would be an additional parameter to the task to pass in the resolved targeting packs and paths.  The SearchPaths parameter would also support an additional search path for the targeting packs.  There would be an additional resolver used by ResolveAssemblyReference which would resolve simple name References to the reference assemblies in the targeting pack.
+
+### Conflict Resolution
+
+Conflict resolution runs before ResolveAssemblyReference, however there may be conflicts involving references that will be resolved from a targeting pack by ResolveAssemblyReference.  If there is a simple name reference to an assembly which is in multiple targeting packs, then ResolveAssemblyReference needs to pick the best one (using similar logic to conflict resolution).  If there is a conflict between a simple name which will be resolved by ResolveAssemblyReference and another reference (either from a NuGet package or that has been resolved from a targeting pack already by ResolveTargetingPackAssets), then ResolvePackageFileConflicts needs to resolve the conflict.  It already does something similar for .NET Framework, by reading the FrameworkList to get the version information that a reference will resolve to.
+
+### NuGet and transitivity
+
+If default references for a FrameworkReference are disabled (ie `IncludeReferences="False"`), then the FrameworkReference needs to flow transitively (through projects and NuGet packages) in order for the right information to be written to the runtimeconfig.json.  However, the compilation references from that FrameworkReference should not flow transitively.  Possible ways to achieve this would include:
```

Comment:
>the compilation references from that FrameworkReference should not flow transitively.  

Asking for my edification:  Why not? Or rather, why shouldn't this be available as an option?

Reply:
If you want to reference a single assembly from a targeting pack, you probably don't want all your consumers to reference all the assemblies from the targeting pack just because they referenced your project.